swap_meet/item.py

```diff
@@ -1,2 +1,22 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, age = 0, category = "", condition = 0):
+        self.category = category
+        self.condition = condition
+        self.age = age
+        
+# Wave 3       
+    def __str__(self):
+        return "Hello World!"
+
+# Wave 5
+    def condition_description(self):
+        condition_descriptions = ["I dont like zero", "I dont like one either", 
+        "Maybe I like two", "Hmmmm.....probably three is nice", "I like four",
+        "Five is a nice number"]
+        return condition_descriptions[self.condition]
```

Comment:
This works if the condition is an integer, but what would happen if it were a float? (Some of the test data shows the condition as `3.5`. How could you change this method to accommodate that?